code/datums/diseases/chronic_illness.dm

```diff
@@ -0,0 +1,103 @@
+/datum/disease/chronic_illness
+	name = "Hereditary Manifold Sickness"
+	max_stages = 5
+	spread_text = "Unspread Illness"
+	spread_flags = DISEASE_SPREAD_NON_CONTAGIOUS
+	disease_flags = CHRONIC
+	process_dead = TRUE
+	stage_prob = 0.25
+	cure_text = "Sansufentanyl"
+	cures = list(/datum/reagent/medicine/sansufentanyl)
+	infectivity = 0
+	agent = "Quantum Entanglement"
+	viable_mobtypes = list(/mob/living/carbon/human)
```

Comment:
IIRC, you might also need to expand the infectable_biotypes, although I could be wrong here

might also need bypasses_immunity = TRUE?

Reply:
It's non_contagious, I'll check it out though. It shouldn't need bypass immunity because it's incurable. I'll write it up though.